packages/Testing/PHPUnit/PlatformAgnosticAssertions.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Testing\PHPUnit;
+
+use Nette\Utils\FileSystem;
+use PHPUnit\Framework\Constraint\IsEqual;
+
+/**
+ * Relaxes phpunit assertions to be forgiving about platform issues, like directory-separators or newlines.
+ */
+trait PlatformAgnosticAssertions
```

Comment:
Maybe some mention of Windows/Linux/MacOS would be useful with specific differences that this trait fixes.
So we know where exact places we should look for bugs in the future.